app/ui/browser/views/navbar/location-completion-row.jsx

```diff
@@ -13,29 +13,56 @@ specific language governing permissions and limitations under the License.
 import React, { PropTypes } from 'react';
 
 import Style from '../../../shared/style';
+import { prettyUrl } from '../../../../shared/url';
+
+const GRADIENT_MASK = `-webkit-linear-gradient(left,
+                      rgba(0, 0, 0, 1) 0%,
+                      rgba(0, 0, 0, 1) 90%,
+                      rgba(0, 0, 0, 0) 95%)`;
 
 const ROW_STYLE = Style.registerStyle({
-  // We're nesting multi-line items inside a flexbox, so
-  // we need to mark these children as columnar.
-  flexDirection: 'column',
+  display: 'flex',
+  padding: '5px',
+});
+
+const TITLE_STYLE = Style.registerStyle({
+  overflow: 'hidden',
+  whiteSpace: 'nowrap',
+  paddingRight: '10px',
+  flex: 6,
+  '-webkit-mask-image': GRADIENT_MASK,
```

Comment:
Looks like only the prefixed version is supported, and same with the linear-gradient, as use by the mask-image :\